bin/deploy.sh

```diff
@@ -30,21 +30,11 @@ source ./borrower/deploy.sh
 
 # finalize deployment
 message Finalize Deployment
-GOVERNANCE="0x$ETH_FROM"
 
 seth send $ROOT_CONTRACT 'prepare(address,address,address)' $LENDER_DEPLOYER $BORROWER_DEPLOYER $GOVERNANCE
 seth send $ROOT_CONTRACT 'deploy()'
 
 success_msg "Tinlake Deployment Finished"
 success_msg "Deployment File: $(realpath $DEPLOYMENT_FILE)"
 
-#touch $DEPLOYMENT_FILE
-addValuesToFile $DEPLOYMENT_FILE <<EOF
```

Comment:
I think we should add it the deployment json file to know for each deployment who is in control

Reply:
it is added, but already in the setup_local_config_file